test/domain/user/userControllerTest.js

```diff
@@ -0,0 +1,194 @@
+require('rootpath')();
+
+const sinon          = require('sinon');
+const chai           = require('chai');
+const expect         = chai.expect;
+const chaiAsPromised = require("chai-as-promised");
+
+chai.use(chaiAsPromised);
+
+const HttpStatus = require('http-status-codes');
+
+const NotFoundError  = require('error/notFoundError');
+const UserService    = require('domain/user/userService');
+const UserController = require('domain/user/userController');
+
+describe('UserController', function() {
+
+    let sandbox;
+
+    let userController;
+
+    before(async function() {
+        const userService = new UserService();
+        userController    = new UserController(userService);
+    });
+
+    beforeEach(function() {
+        sandbox = sinon.createSandbox();
+    });
+
+    afterEach(function() {
+        sandbox.restore();
+    });
+
+    describe('getAll', function() {
+
+        it('should return all users', async function() {
+            const getAllStub = sandbox.stub(UserService.prototype, 'getAll').callsFake(() => Promise.resolve([{name: 'User name'}]));
+
+            const request  = {};
+            const response = {
+                status: sinon.fake(() => response),
+                json:   sinon.fake(() => response),
+            };
+
+            await userController.getAll(request, response);
+
+            expect(getAllStub.calledOnce).equal(true);
```

Comment:
Take a look at 'sinon-chai' plugin for better assertions when validating sinon mocks